src/frontends/pytorch/src/op/div.cpp

```diff
@@ -90,6 +90,17 @@ OutputVector translate_div_fx(const NodeContext& context) {
     return translate_div_common(context, x, y, rounding_mode, false);
 };
 
+OutputVector translate_div_fx_(const NodeContext& context) {
```

Comment:
Where is this used?

Reply:
Missed committing the changes in op_table.cpp. Thanks for catching it maxim